ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -303,7 +331,9 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
           trigger={<Label horizontal size="mini" content={previousCall.content} color={previousCall.color} />}
         />
       )}
-      {`${genotype.gq || genotype.qs || '-'}${variant.svType ? '' : genotype.numAlt >= 0 && `, ${genotype.ab ? genotype.ab.toPrecision(2) : '-'}`}`}
+      {`${genotype.gq || genotype.qs || '-'}${
+        variant.svType ? '' : genotype.numAlt >= 0 && `, ${genotype.ab ? genotype.ab.toPrecision(2) :
+          `${genotype.hl ? genotype.hl.toPrecision(2) : '-'}`}`}`}
```

Comment:
this is really hard to read now. I think this might be cleaner to do the followig:
```
let secondaryQuality
const showSecondaryQuality = !variant.svType && genotype.numAlt >= 0
if (showSecondaryQuality) {
  secondaryQuality = genotype.ab || genotype.hl
}
...
const content = (
...
  {genotype.gq || genotype.qs || '-'}
  {showSecondaryQuality && `, ${secondaryQuality ? secondaryQuality.toPrecision(2) : '-'}`}
...
)
```